app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts
+
```

Comment:
Hi, we are missing the relationship between `user` and `comments`, let us add `has_many :comments`